sdk/tables/azure-data-tables/azure/data/tables/_table_client.py

```diff
@@ -54,13 +39,12 @@ class TableClient(TablesBaseClient):
     :ivar str url: The full URL to the Tables account.
     """
 
-    def __init__(  # pylint: disable=missing-client-constructor-parameter-credential
+    def __init__( # pylint: disable=missing-client-constructor-parameter-credential
         self,
-        endpoint,  # type: str
-        table_name,  # type: str
-        **kwargs  # type: Any
-    ):
-        # type: (...) -> None
+        endpoint: str,
+        table_name: str,
+        **kwargs
```

Comment:
Could we add `credential` here using the keyword-only syntax, like we do for the async TableClient? That should improve the APIview.

Reply:
Have changed to be the same in async client.